napari/layers/shapes/_shapes_utils.py

```diff
@@ -426,27 +426,31 @@ def find_corners(data):
     return corners
 
 
-def center_radii_to_corners(center, radii):
+def center_radii_to_corners(
+    center: npt.NDArray, radii: npt.NDArray
+) -> npt.NDArray:
     """Expands a center and radii into a four corner rectangle
 
     Parameters
     ----------
-    center : np.ndarray | list
-        Length 2 array or list of the center coordinates
-    radii : np.ndarray | list
-        Length 2 array or list of the two radii
```

Comment:
Looking at the logic below, this wouldn't work with a list, so I've removed that option from the docstring.

Reply:
yeah, subtracting lists is 👀